src/element/textWysiwyg.tsx

```diff
@@ -520,15 +516,6 @@ export const textWysiwyg = ({
             }),
           });
         }
-      } else {
-        mutateElement(container, {
-          boundElements: container.boundElements?.filter(
-            (ele) =>
-              !isTextElement(
-                ele as ExcalidrawTextElement | ExcalidrawLinearElement,
-              ),
-          ),
-        });
```

Comment:
wait, this we still want? If you delete bound text this way that already exists.

Reply:
no this is not needed since its taken care here - https://github.com/excalidraw/excalidraw/blob/master/src/element/binding.ts#L702

Reply:
`fixBindingsAfterDeletion` doesn't handle this case:

1. create sticky note
2. delete text by making it empty
3. confirm

the container's `boundElements` is not updated